dask_ml/preprocessing/_block_transformer.py

```diff
@@ -0,0 +1,76 @@
+import dask.array as da
+import dask.dataframe as dd
+import pandas as pd
+from sklearn.base import BaseEstimator, TransformerMixin
+
+from dask_ml.utils import check_array, handle_zeros_in_scale
+
+
+class BlockTransformer(BaseEstimator, TransformerMixin):
+    """Construct a transformer from a an arbitrary callable
+
+    The BlockTransformer forwards the blocks of the X arguments to a user-defined
+    callable and returns the result of this operation.
+    This is useful for stateless operations, that can be performed on the cell or
+    block level, such as taking the log of frequencies. In general the transformer
+    is not suitable for e.g. standardization tasks as this requires information for
+    a complete column.
+
+    Parameters
+    ----------
+    func : callable
+        The callable to use for the transformation.
+
+    validate : bool, optional default=False
+         Indicate that the input X array should be checked before calling
+        ``func``.
+
+    kw_args : dict, optional
+        Dictionary of additional keyword arguments to pass to func.
+
+    Examples
+    --------
+
+    >>> import dask.datasets
+    >>> import pandas as pd
+    >>> df = dask.datasets.timeseries()
+    >>> df
```

Comment:
This should be like

```
>>> import dask.datasets
>>> import pandas as pd
>>> df = dask.datasets.timeseries()
>>> df  # doctest: +SKIP
Dask DataFrame Structure:
                   id    name        x        y
npartitions=30
2000-01-01      int64  object  float64  float64
2000-01-02        ...     ...      ...      ...
...               ...     ...      ...      ...
2000-01-30        ...     ...      ...      ...
2000-01-31        ...     ...      ...      ...
Dask Name: make-timeseries, 30 tasks
```

Basically, the input *and output* of a python session.

May also want `from dask_ml.preprocessing import BlockTransformer` at the start of the example.

Reply:
is there a way to quickly build the documentation and maybe only parts of it?

Initially I added the output as you did but the dockets just failed. (didn't use skip though).